### PR TITLE
google sign in Fix

### PR DESCRIPTION
**Pull request by pinocchio-life-like**

### CODE REVIEW

**Review by andrew-bierman — APPROVED** [review]
lgtm

File: packages/app/components/dashboard/FeedPreview.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import useTheme from '../../hooks/useTheme';
 import Carousel from '../carousel';
 import useCustomStyles from 'app/hooks/useCustomStyles';
 import { useFeed } from 'app/hooks/feed';
+import { background } from 'native-base/lib/typescript/theme/styled-system';
```
Comment (andrew-bierman) [inline packages/app/components/dashboard/FeedPreview.tsx:9]: cleanup

Comment (pinocchio-life-like) [reply packages/app/components/dashboard/FeedPreview.tsx:9]: Ok

Comment (andrew-bierman) [reply packages/app/components/dashboard/FeedPreview.tsx]: thanks, this is approved to merge

**Comment by andrew-bierman** [comment]
is this working for you ? I'm getting the below error locally 

```typescript
Access blocked: Authorization Error

The OAuth client was not found.
If you are a developer of this app, see [error details]
Error 401: invalid_client
```

**Comment by pinocchio-life-like** [comment]
> is this working for you ? I'm getting the below error locally
> 
> ```ts
> Access blocked: Authorization Error
> 
> The OAuth client was not found.
> If you are a developer of this app, see [error details]
> Error 401: invalid_client
> ```

This was because our CF is using vite for preview, I have added a env var for google ID in CF env variables for preview and it is okay now.

**Pull request merged**